config.json

```diff
@@ -1,8 +1,11 @@
 {
-    "name_prefix": "guy6",
+    "name_prefix": "surface",
     "chunking": {
         "chunk_size": [1000],
-        "overlap_size": [200]
+        "overlap_size": [200],
+        "generate_title": false,
```

Comment:
Do these need to be added if they are setting defaults?
Is the idea to add the defaults to help make them discoverable?

Reply:
great question, I continued with this notion, I guess yes, the Idea to make them discoverable.

Reply:
do we want to change this notion? if so I think we should do it in separate PR...